.docker/backend.Dockerfile

```diff
@@ -9,13 +9,14 @@ COPY ./package*.json ./
 COPY ./tsconfig.json ./
 COPY ./.eslintrc.yml ./
 COPY ./shared ./shared/
-COPY ./backend ./backend/
+COPY ./backend/core ./backend/core
 
-RUN npm ci -w shared -w backend
+RUN npm ci -w shared -w backend/core
+RUN npm run postinstall:backend
```

Comment:
Why do you need manually run postinstall script?

Reply:
I`m not sure wht but when running postinstall in socket.Dockerfile it throws this error 
![image](https://user-images.githubusercontent.com/56759981/184426788-466ed793-549d-4571-960b-acc203c94af6.png)